netuitive/conf/collectors/TCPCollector.conf

```diff
@@ -0,0 +1 @@
+enabled = True
```

Comment:
Shouldn't this collector be disabled by default?

Reply:
Yes, good catch.

Reply:
Updated!